### PR TITLE
Feature deployment configuration

### PR DESCRIPTION
**Pull request by AangelF**
# Description

Add necessary files to deploy the website on Render.com and edit the configuration on other files.
The files edited were:
- puma.rb
- production.rb

The files added were:
- render-build.sh
- **database.yml** (this file is necessary for production).

### CODE REVIEW

File: app/views/admins/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    </div>
+  <% end %>
+  <%= bootstrap_form_for(@login_admin) do |f| %>
+  <div class="form-floating">
+    <%= f.text_field :admin, class: "form-control", placeholder: "ID de Administrador", hide_label: true %>
+    <%= f.password_field :privateNumber, class: "form-control", placeholder: "Numero Privado", hide_label: true %>
+    <%= f.submit "Log In", class: "w-100 btn btn-lg btn-primary" %>
+  </div>
+  <% end %>
+</main>
```
Comment (juperameza) [inline app/views/admins/index.html.erb:18]: Add a new line at the end of this file

File: bin/render-build.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# exit on error
+set -o errexit
+
+bundle install
+bundle exec rake assets:precompile
+bundle exec rake assets:clean
+bundle exec rake db:migrate
```
Comment (juperameza) [inline bin/render-build.sh:8]: New line at the end of this file

File: db/seeds.rb [hunk 3]
```diff
@@ -7,3 +7,4 @@
 #   Character.create(name: "Luke", movie: movies.first)
 store = Store.create(name: 'Oxxo', address: '123 Street')
 empleado = Employee.create(name: 'Bart Simpson', email: 'bart@simpson.com', position: 'regional manager', statusActive: true, store_id: 1, privateNumber: 123456)
+admin = Admin.create(admin: 1, privateNumber: 123456)
```
Comment (juperameza) [inline db/seeds.rb:10]: New line at the end of this file

File: app/controllers/admins_controller.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+  def create
+    @login_admin = Admin.new(loginadmin_params)
+    @login_admin.save ? (redirect_to root_path, success: "login de admin exitoso") : (redirect_to admin_path, danger: "ERROR: no login de admin")
+  end
+
+  private
+
+  def loginadmin_params
+    params.require(:admin).permit(:admin, :privateNumber)
```
Comment (juperameza) [inline app/controllers/admins_controller.rb:11]: I recommend you use devise gem but this is ok

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/helpers/logins_admins_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module LoginsAdminsHelper
```
Comment (oscaregr) [inline app/helpers/logins_admins_helper.rb:1]: if you no use this remove

File: app/helpers/admins_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module AdminsHelper
+end
```
Comment (oscaregr) [inline app/helpers/admins_helper.rb:2]: same here, if you no use this remove

File: app/controllers/admins_controller.rb [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  end
+
+end
```
Comment (oscaregr) [inline app/controllers/admins_controller.rb:14]: add one line blanck, here

File: app/views/admins/index.html.erb (hunk 1 above)
Comment (oscaregr) [inline app/views/admins/index.html.erb:18]: add one line blanck

**Review by oscaregr — CHANGES_REQUESTED** [review]

**Review by xManuel007 — APPROVED** [review]

File: app/views/admins/index.html.erb (hunk 1 above)
Comment (oscaregr) [inline app/views/admins/index.html.erb:18]: add line blanck , and be sure, open the file in github and check blanck is add

File: app/controllers/admins_controller.rb (hunk 7 above)
Comment (oscaregr) [inline app/controllers/admins_controller.rb:14]: add line blanck under the last end

**Review by oscaregr — CHANGES_REQUESTED** [review]
do this, and i made the approve

File: test/models/administrator_test.rb [hunk 8]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class AdministratorTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (dagp-ucol) [inline test/models/administrator_test.rb:6]: Hey, remember to delete unused code when generating files, controllers, models, etc.

File: test/models/admin_test.rb [hunk 9]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class AdminTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (dagp-ucol) [inline test/models/admin_test.rb:6]: Hey, remember to delete unused code when generating files, controllers, models, etc.

**Review by dagp-ucol — APPROVED** [review]
I see that you resolved some other issues.
Be careful with good practices.

I know this comments may be frustrating, but the objective of the bootcamp is to ensure that you are learning but also with good practices.

PR looks good to me, just be careful.

**Review by oscaregr — APPROVED** [review]
LGTM

**Review by juperameza — APPROVED** [review]
Lgtm

**Pull request merged**